### ISSUE TITLE
[BE] Expand backend to accept attachements

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#104]
Expand backend so that attachements sent by the user via the frontend are accepted and stored in the database together with the respective ticket.

## Acceptance criteria
- [ ] attachement is stored together with the correct ticket in the database
- [ ] attachement is accepted without an error
- [ ] at least pdfs and pngs / jpgs can be stored

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by M-HRL** [comment]
@garvinkon @Gnagniko 

### DONE

- Added new endpoint
- Refactored the api backend to fit standards and for attachments
- Adapted the old tests

### TODO:

- Adding additional tests
- Adapting the fronted to the new text endpoint
